### ISSUE TITLE
Provide feedback to the speakers

### ISSUE THREAD

**Issue by ascheman** [dukecon/dukecon#43]
As a conference participant I would like to give feedback to the speaker to allow the person to improve her talk or just to let her know I liked the presentation (cf. #41). The feedback should be forwarded anonymously to the speaker if not checked otherwise. I am aware that the feedback can be edited or completely oppressed by the conference staff (#42) if the feedback could be understood in an insulting way.

**Comment by ckeimel** [comment]
As a conference participant I would like to see in the web app / mobile app which sesssions I have provided feedback to. As it is at the moment I have no idea if the feedback was even saved. 

It would also be great if I could review and change my feedback. Often, when I write feedback I hit save and 5 minutes later I have an idea of something I would like to add. As it is now, I could write a 2nd feedback (well .. the button is there and enabled, haven't had the curage to press it). But I do not know: Will that be a new feedback or will it overwrite my last feedback. Or will I get an error?

Basically: I would like some feedback on my trying to give feedback :-)

**Comment by sippsack** [comment]
@ckeimel: Thanks for your ideas. We will try to improve the feedback experience for the user in the near future and some of your suggestions are already on our backlog. Feel free to join us and help improving this feature.
